librosa/core/constantq.py

```diff
@@ -860,13 +860,18 @@ def vqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84, gamma=None,
 
     len_orig = len(y)
 
+    alpha = (2.0**(1. / bins_per_octave) - 1)
+
     if fmin is None:
         # C1 by default
         fmin = note_to_hz('C1')
 
     if tuning is None:
         tuning = estimate_tuning(y=y, sr=sr, bins_per_octave=bins_per_octave)
 
+    if gamma is None:
+        gamma = 24.7 * alpha / 0.108
```

Comment:
Please document these constants, with a reference to a paper that explains it (ideally with URL too)